geode-connectors/src/acceptanceTest/java/org/apache/geode/connectors/jdbc/JdbcAsyncWriterIntegrationTest.java

```diff
@@ -173,17 +172,17 @@ public void canDestroyFromTable() throws Exception {
   @Test
   public void canDestroyFromTableWithCompositeKey() throws Exception {
     setupRegion("id,age");
-    JSONObject compositeKey1 = new JSONObject();
-    compositeKey1.put("id", pdxEmployee1.getField("id"));
-    compositeKey1.put("age", pdxEmployee1.getField("age"));
-    JSONObject compositeKey2 = new JSONObject();
-    compositeKey2.put("id", pdxEmployee2.getField("id"));
-    compositeKey2.put("age", pdxEmployee2.getField("age"));
-    employees.put(compositeKey1.toString(), pdxEmployee1);
-    employees.put(compositeKey2.toString(), pdxEmployee2);
+    PdxInstance compositeKey1 = cache.createPdxInstanceFactory("IdAgeKeyType").neverDeserialize()
```

Comment:
My preference would be to keep with less negative statements. I feel like `.setDeserializable(false)` corresponds better with `isDeserializable`.

Reply:
I think "neverDeserialize" is better because it does not take a boolean argument. I like that it is more "opinionated".